MetaLamp/nft-marketplace/src/Plutus/Contracts/Services/Auction/StateMachine.hs

```diff
@@ -151,7 +152,8 @@ auctionTransition getAdditionalPayoutConstraints params@Auction{..} state@State{
                 constraints =
                     Constraints.mustValidateIn (Interval.to aEndTime) -- While the auction hasn't ended,
                     <> Constraints.mustPayToPubKey highestBidder (Ada.toValue highestBid) -- and the highest bidder the asset
-                    <> Constraints.mustPayToPubKey aOwner aAsset -- and the highest bidder the asset
+                    <> Constraints.mustPayToPubKey aOwner (aAsset + minAdaTxOutValue) -- and the highest bidder the asset
+                    -- TODO: is it okay that buyer receive additional 2ADA? Should we initially add them to the bid price?
```

Comment:
I think the user who started the auction should receive this value back (it serves as a kind of pledge)

Reply:
Absolutely, I will revise payment algorithms in new task. Here we haven't ability to send NFT without minimal Ada, it's a reason why `minAdaTxOutValue` go to the buyer. 
This task is just about update plutus. New logic will be in the new PR :)